src/util/HeaderUtil.ts

```diff
@@ -357,6 +362,16 @@ export const parseAcceptLanguage = (input: string): AcceptLanguage[] => {
   return results;
 };
 
+/**
+ * Parses an Accept-DateTime header string.
+ *
+ * @param input - The Accept-DateTime header string.
+ *
+ * @returns An array with a single {@link AcceptDatetime} object.
+ */
+export const parseAcceptDateTime = (input: string): AcceptDatetime[] =>
+  [{ range: input, weight: 1 }];
```

Comment:
Is there actually a valid format for the datetime headers? ISO 8601? Because then we could also check that here (since all the other functions here also check for valid formatting).

Reply:
In be121e2074f634b683f347cc2a7a3fb8f0a230d3. Also see https://github.com/solid/community-server/issues/469.

I wanted to also _parse_ the date, but it is supposed to be a (validated) string. Fair enough though.